### PR TITLE
[1LP][RFR] Manual vm reconfigure cleanup

### PR DESCRIPTION
**Pull request by niyazRedhat**
Purpose or Intent
=================

1. Moved my tests from manual.py to my vm reconfigure test files

### CODE REVIEW

File: cfme/tests/infrastructure/test_vm_reconfigure.py [hunk 1]
```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
 
     # Revert back to original config
     reconfigure_vm(small_vm, orig_config)
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.parametrize('change_type', ['sockets', 'memory'])
+def test_vm_reconfig_add_remove_hw_hot_vmware():
```
Comment (digitronik) [inline cfme/tests/infrastructure/test_vm_reconfigure.py]: @niyazRedhat: `travis` is failing due to this.... you are trying to parametrize manual test but `change_type` not used.

```
def test_vm_reconfig_add_remove_hw_hot_vmware(change_type):
```

File: cfme/tests/infrastructure/test_vm_reconfigure.py [hunk 2]
```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
...
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.parametrize('disk_type', ['thin', 'thick'])
+@pytest.mark.parametrize(
+    'disk_mode', ['persistent', 'independent_persistent', 'independent_nonpersistent'])
+def test_vm_reconfig_add_remove_disk_hot():
```
Comment (digitronik) [inline cfme/tests/infrastructure/test_vm_reconfigure.py]:
```
def test_vm_reconfig_add_remove_disk_hot(disk_type, disk_mode):
```

File: cfme/tests/infrastructure/test_vm_reconfigure.py [hunk 3]
```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
...
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.parametrize('disk_type', ['thin', 'thick'])
+@pytest.mark.parametrize(
+    'disk_mode', ['persistent', 'independent_persistent', 'independent_nonpersistent'])
+def test_vm_reconfig_resize_disk_cold():
```
Comment (digitronik) [inline cfme/tests/infrastructure/test_vm_reconfigure.py]: same as above

File: cfme/tests/infrastructure/test_vm_reconfigure.py [hunk 4]
```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
...
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.parametrize('disk_type', ['thin', 'thick'])
+@pytest.mark.parametrize(
+    'disk_mode', ['persistent', 'independent_persistent', 'independent_nonpersistent'])
+def test_vm_reconfig_resize_disk_hot():
```
Comment (digitronik) [inline cfme/tests/infrastructure/test_vm_reconfigure.py]: same as above parametrization

File: cfme/tests/infrastructure/test_vm_reconfigure.py [hunk 5]
```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
...
+@pytest.mark.parametrize('change_type', ['sockets', 'memory'])
+def test_vm_reconfig_add_remove_hw_hot_vmware(change_type):
+    """
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: high
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
```
Comment (lina-nikiforova) [inline cfme/tests/infrastructure/test_vm_reconfigure.py]: should be `caseautomation: notautomated` here and in all other places
see https://github.com/ManageIQ/integration_tests/blob/master/conf/polarion_tools.yaml#L88

File: cfme/tests/infrastructure/test_vm_reconfigure.py [hunk 6]
```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
...
+
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: high
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
+        testtype: functional
+        startsin: 5.10
```
Comment (lina-nikiforova) [inline cfme/tests/infrastructure/test_vm_reconfigure.py]: if it starts in `5.10`, there should be a decorator `@pytest.mark.ignore_stream('5.9')`

File: cfme/tests/infrastructure/test_vm_reconfigure.py [hunk 7]
```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
...
+        startsin: 5.6
+        casecomponent: infra
+        testSteps:
+            1. get new configured appliance ->add vmware provider
+            2. provision 2 new vms
+            3. power off 1 vm -> select both vms
+            4. configure-->reconfigure vm
+            5. increase/decrease counts
+            6. power on vm
+            7. check changes
```
Comment (lina-nikiforova) [inline cfme/tests/infrastructure/test_vm_reconfigure.py]: I think you should keep the original test steps and expected results somewhere, otherwise it's not clear what to expect from this changes

```
        testSteps:	
            1. Hot increase	
            2. Hot Decrease	
            3. Cold Increase	
            4. Cold Decrease	
            5. Hot + Cold Increase	
            6. Hot + Cold Decrease	
        expectedResults:	
            1. Action should fail	
            2. Action should fail	
            3. Action should succeed	
            4. Action should succeed	
            5. Action should fail	
            6. Action should Error
```

File: cfme/tests/test_manual.py [hunk 8]
```diff
@@ -3358,20 +3284,6 @@ def test_host_tagged_crosshair_op_vsphere55():
     pass
 
 
-@pytest.mark.manual
-@test_requirements.reconfigure
-@pytest.mark.tier(1)
-def test_vm_reconfig_add_remove_network_adapters_vsphere67_nested_vmkernel():
```
Comment (lina-nikiforova) [inline cfme/tests/test_manual.py]: why did you remove this test as well as other 3 `test_vm_reconfig_add_remove_network_adapters_vsphere67_nested*`? Or was it renamed?

File: cfme/tests/infrastructure/test_vm_reconfigure.py [hunk 9]
```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
...
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: medium
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
+        testtype: functional
+        startsin: 5.6
+        casecomponent: infra
+        testSteps:
```
Comment (lina-nikiforova) [inline cfme/tests/infrastructure/test_vm_reconfigure.py]: same, maybe original test steps/expected results would help

**Comment by jawatts** [comment]
Outstanding issues, moving to WIPTEST

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/test_manual.py (hunk 8 above)
Comment (niyazRedhat) [reply cfme/tests/test_manual.py]: Yes I renamed it --> ```def test_vm_reconfig_add_remove_network_adapters(adapters_type):```

File: cfme/tests/infrastructure/test_vm_reconfigure.py (hunk 5 above)
Comment (digitronik) [reply cfme/tests/infrastructure/test_vm_reconfigure.py]: As per our new updates. we are avoiding this `key`. `casautomation` taken care by `tier` marker.
